Edit transcript md4cei couey 10.22.24

solarfire9 opened the pull request:
Edited for spelling,  names, punctuation, clarity.

Soothspider: Not sure if this is showing up where it's supposed to. Please let me know if you can see this pull request. Thanks!

Commit:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

done to line 973

Commit:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

1:15:49 Lukach

Commit:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

Finished edit

Commit:
Merge branch 'gigaohm:main' into Edit-MD4CEI-Couey-10.22.24

soothspider commented:
We don't need to do another PR here. We can just do the one at (instead): https://github.com/gigaohm/Stream.Transcripts/pull/4 and once it is merged, I can sync this repo and it'll just get it automatically.

soothspider committed:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

Small edit to speaker names.

soothspider committed:
Update v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

Attempted easy fix of no timestamps. Removing whitespace for spurious new lines to see if captions/subtitles will render as 1 block.

ghost commented:
I learned a lot this time around...thanks for your patience and support! I still have a lot to learn.
